### DISCUSSION TITLE
Datenbank Fotos/Grafiken Fitnessübungen

### DISCUSSION THREAD

**Discussion by therealchrismcfly** [neuefische/questions#118] [Web] [answered]
Hallo zusammen,

weiß jemand von einer Datenbank mit einer Auswahl an Fotos/Grafiken von verschiedenen Fitnessübungen bzw. gab es da in der Vergangenheit Capstone Projekte zu? Wichtig wäre mindstens 5 einheitliche Bilder (deswegen ist es mit Pexels, Unsplash & Co etwas schwierig).

Danke schonmal :-)

**Comment by mpagels** [comment]
Ginge sowas?
https://www.shutterstock.com/de/image-vector/woman-workout-fitness-aerobic-exercises-vector-496580704

**Reply by therealchrismcfly** [reply]
Ja, auf jeden Fall. Aber in kostenlos halt :)

**Comment by alexclaes** [comment, marked as answer]
Ich habe mal kurz nach Open Source Workout Apps recherchiert. Bin auf Apps gestoßen, die Assets im Quellcode haben und diese auch in der Lizenz erwähnen. Vielleicht ist da ja was dabei.

https://github.com/chaosbastler/opentraining/tree/master/app/assets/opentraining-exercises

https://github.com/oliexdev/openWorkout/tree/master/app/src/main/assets/image

https://gitlab.com/enjoyingfoss/feeel/-/tree/master/assets

**Reply by therealchrismcfly** [reply]
Vielen Dank für die Tipps :-) - ich bin bei Adobe Stock fündig geworden. Da kann man ein Testabo abschließen und 10 Bilder umsonst bekommen (falls jemand in der Zukunft auf Suche ist).
